lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/experianKbvCri/ContractTest.java

```diff
@@ -0,0 +1,1041 @@
+                    {
+                      "checkDetails": [
+                        {
+                          "checkMethod": "kbv",
+                          "kbvQuality": 2,
+                          "kbvResponseMode": "free_text"
+                        },
+                        {
+                          "checkMethod": "kbv",
+                          "kbvQuality": 2,
+                          "kbvResponseMode": "multiple_choice"
+                        },
+                        {
+                          "checkMethod": "kbv",
+                          "kbvQuality": 1,
+                          "kbvResponseMode": "multiple_choice"
+                        }
+                      ],
+                      "verificationScore": 2,
+                      "txn": "dummyTxn",
+                      "type": "IdentityCheck"
+                    }]
+                  }
+                }
+                """;
+
+    private static final String VALID_VC_BODY_WITH_WRONG_ANSWER =
+            """
+            {
+               "iss": "dummyExperianKbvComponentId",
+               "sub": "test-subject",
+               "nbf": 4070908800,
+               "vc": {
+                 "type": [
+                   "VerifiableCredential",
+                   "IdentityCheckCredential"
+                 ],
+                 "credentialSubject": {
+                   "name": [
+                     {
+                       "nameParts": [
+                         {
+                           "type": "GivenName",
+                           "value": "Mary"
+                         },
+                         {
+                           "type": "FamilyName",
+                           "value": "Watson"
+                         }
+                       ]
+                     }
+                   ],
+                   "birthDate": [
+                     {
+                       "value": "1932-02-25"
+                     }
+                   ],
+               "address": [
+                     {
+                       "uprn": "10022812929",
+                       "organisationName": "FINCH GROUP",
+                       "subBuildingName": "UNIT 2B",
+                       "buildingNumber": "16",
+                       "buildingName": "COY POND BUSINESS PARK",
+                       "dependentStreetName": "KINGS PARK",
+                       "streetName": "BIG STREET",
+                       "doubleDependentAddressLocality": "SOME DISTRICT",
+                       "dependentAddressLocality": "LONG EATON",
+                       "addressLocality": "GREAT MISSENDEN",
+                       "postalCode": "HP16 0AL",
+                       "addressCountry": "GB"
+                     }
+                   ]
+             },
+             "evidence": [
+                 {
+                   "type": "IdentityCheck",
+                   "txn": "dummyTxn",
+                   "verificationScore": 0,
```

Comment:
I know that this example has come from Confluence, but I'm suspicious of this verification score as this should be a successful VC. This table suggest that the score should be 2 in this case: https://govukverify.atlassian.net/wiki/spaces/OJ/pages/3580100670/Experian+KBV+question+policy+-+WIP#Current-Implementation-(with-Contra-Indicators)

Reply:
Yeah a KBV fail would be zero scoring with a V03 CI (the one below)
But there is another scenario - Experian KBV uses a zero scoring VC without a CI to denote abandon/thin file

Reply:
That sounds like an additional scenario worth testing, this VC has too many questions to be thin file or abandoned though :)